integrations/graal/mp-native-image-extension/pom.xml

```diff
@@ -61,6 +61,11 @@
             <artifactId>svm</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Why is this change in?

Reply:
This was added to make `ClassFileUtils` block be uncommented and build successfully. But as you mentioned below, that will fail with native build because  org.jboss.weld.util.bytecode.ClassFileUtils does not exist anymore. So if that block is not needed anymore, then I can remove this as well.

Reply:
As part of the change to remove internal class ClassFileUtils from WeldSubstitutions, this dependency was removed as well.